src/libs/actions/Task.ts

```diff
@@ -605,7 +613,7 @@ function editTask(report: OnyxTypes.Report, {title, description}: OnyxTypes.Task
 
     const parameters: EditTaskParams = {
         taskReportID: report.reportID,
-        title: reportName,
+        htmlTitle: reportName,
```

Comment:
Does backend already support `htmlTitle`? @roryabraham Can you confirm? and is this tested on staging as well as prod APIs?